npm/npm_import.bzl

```diff
@@ -302,6 +303,9 @@ def npm_translate_lock(
 
             See https://github.com/bazelbuild/bazel/issues/8106
 
+        verify_patches: Verify that all patches in the folder appear in the patches attribute. Does not recurse
```

Comment:
Oh. Should specify that this takes a path to a folder in the WORKSPACE. I think it would be confusing to take a label actually since labels to folders that are also packages are not valid.